amnestywebsite/humanity-theme
# Replace page.php with FSE template

Ref: https://github.com/amnestywebsite/humanity-theme/issues/370

Adds a new html page template comprising of patterns for article, sidebar and header and removes the old php page template.

Also includes type stylesheets in the editor

**Steps to test**:
1. View a page on the front end and check it looks as expected
2. Enter the site editor and look at the page template and ensure that matches the front end and looks as expected

Example:
http://bigbite.im/i/MVzZA1
http://bigbite.im/i/ZHgd85
